config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
   
+  get 'likes/create'
+  get 'comments/new'
+  get 'comments/create'
```

Comment:
Are the routes correct as per rails convention?

Reply:
no ma'am , that why i update route but forget to push ma'am.